### PR TITLE
JPERF-449: Expect Docker images to be runnable in CI

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

**Comment by dagguh** [comment]
It shows the problem, which is blocking https://ecosystem.atlassian.net/browse/JPERF-449

**Comment by dagguh** [comment]
I pushed a fix. It works locally. 🤞 for TravisCI

**Comment by dagguh** [comment]
So TravisCI seems to have problems with privileged containers:
```
Error while executing sudo docker run -d  --name jpt-220c358d-538c-4ee4-bebe-f0bb8369d923 hello-world . Exit status code SshResult(exitStatus=125, output=, errorOutput=docker: Error response from daemon: error creating aufs mount to /var/lib/docker/aufs/mnt/74f51ca73ff115982bcad928dde09d6f1b578ed7b83eb8c9a1e195a4a2d5d415-init: mount target=/var/lib/docker/aufs/mnt/74f51ca73ff115982bcad928dde09d6f1b578ed7b83eb8c9a1e195a4a2d5d415-init data=br:/var/lib/docker/aufs/diff/74f51ca73ff115982bcad928dde09d6f1b578ed7b83eb8c9a1e195a4a2d5d415-init=rw:/var/lib/docker/aufs/diff/9facefec9b11fe049ab02d7c2e13103200066c673933010a6266cbe4ef63e2a8=ro+wh,dio,xino=/dev/shm/aufs.xino: invalid argument.

    See 'docker run --help'.

    )
```

CircleCI has no such problems: https://circleci.com/gh/dagguh/infrastructure/13
Here's the [green DockerIT test](https://13-177132082-gh.circle-artifacts.com/0/test-reports/testIntegration/classes/com.atlassian.performance.tools.infrastructure.DockerIT.html).
However, CircleCI doesn't wanna trigger on the atlassian fork.
I can dig deeper, but also I could give GitHub Actions a go. WDYT @mzyromski-atlassian?

**Comment by dagguh** [comment]
So I tried to trigger a CircleCI build, to no avail.
I spiked GitHub actions. They trigger without any problems. And the tests are green: https://github.com/dagguh/infrastructure/runs/953688407?check_suite_focus=true
The only problem is, ChromeIT and its siblings take 30+ minutes to run: [Gradle scan](https://scans.gradle.com/s/7skmfpkhff2k2/tests).

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ jobs:
     - name: Build
       run: ./gradlew build
     - name: Upload test reports
+      if: always()
```
Comment (wyrzyk) [inline .github/workflows/ci.yml:30]: 😕

Comment (dagguh) [reply .github/workflows/ci.yml:30]: Why sad?

**Pull request merged**